src/main/webapp/app/overview/course-lectures/course-lecture-row.component.html

```diff
@@ -16,14 +16,14 @@
                 <h4><fa-icon [icon]="'chalkboard-teacher'"></fa-icon></h4>
             </div>
             <div class="col-sm col">
-                <h4>{{ lecture.title }}</h4>
+                <h4>{{ lecture?.title }}</h4>
```

Comment:
Out of curiosity: What is this change for, since you seem to provide a lecture object (with title) for every test case (except `'should initialize'`)?

Reply:
I think a component should never just throw an error because an input element is not provided, therefor I always fix it when i notice it during initialize. I can revert the change if you want though :)

Reply:
I see, and don't really mind. Fwiw, this would probably still crash in the .component.ts, but again I have no preference either way.